lib/board.rb

```diff
@@ -0,0 +1,57 @@
+class Board
+  attr_accessor :cells
+
+  def initialize()
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(input)
+    return false if (input.to_i <= 0 || input.to_i > 9)
+    @cells[input.to_i - 1]
+  end
+
+  def full? 
+    @cells.any? {|cell| cell == " "} ? false : true
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. 

`cells.none? {|cell| cell == " "}`